README.md

```diff
@@ -111,6 +111,14 @@ mounts:
     # When true, the proxy path defined above gets a suffix to the Path prefix appended to it.
     proxyAppendSuffix: true
 
+    - path: /subdir
+    # When true, all paths starting with this prefix use this mount.
+    pathIsPrefix: true
+    # Provides a path on the host to find the files.
+    # So that BasePath: /tftpboot path: /subdir and client request: /subdir/file.x so that the host
+    # path becomes /tfptboot/subdir/file.x
+    baseDir: /tftpboot
```

Comment:
I'd call it `localDir` to make obvious that files are hosted locally. Although I have also debated `staticDir`.

I'd also prefer the behavior to be slightly less like a "base" path, and more like an "alias". In your example, for a path at `/subdir` it's impossible to use a local directory that does not include `/subdir/` in its path. I'd rather have it work like this: `path: /subdir => localDir: /tftpboot` such that call to `/subdir/test.img` reaches `/tftboot/test/img`. One is still free to make it `localDir: /tftpboot/subdir/` and retain the path component in it.

Reply:
How do you suggest to handle your example? As in the current implementation the following happens:

```
$  http "192.168.1.199:8080/rpi/RPI_EFI-01.fd?spoof=192.168.1.201"
open /home/rock/talos-boot/rpi/rpi/RPI_EFI-01.fd: no such file or directory
```
```
mounts:
  - path: /rpi
    pathIsPrefix: true
    localDir: /home/rock/talos-boot/rpi
```
as you can see `rpi` is added twice. The only way I can think to fix this is to add some logic or flag that checks for such cases unless I'm missing something 🤔

Reply:
You are right, it is necessary to strip the `path` from the original request before mapping it to `localDir`, and I believe the current mounts that "proxy" are currently doing that:
https://github.com/DSpeichert/netbootd/blob/7f8084afc023043d2cc7cb10c7d0a99409f27645/manifest/schema.go#L75-L77

I'd imagine similar handling would be necessary. It also may be useful (although I do agree it may be becoming confusing) to handle the "many to one" vs "many to many" mount type that is controlled by the `proxyAppendSuffix` (in this case, I guess it would be called differently, or we rename it to `appendSuffix`).